src/app/my-regular-giving/my-regular-giving.component.html

```diff
@@ -48,11 +48,9 @@ <h3>
                 </h3>
                 <ul>
                   <li>Amount: {{ mandate.donationAmount | money }}</li>
-                  <!-- @todo-regular-giving consider whether to also show total with match
-                              funds when the next donation will be matched. -->
```

Comment:
I think as this page has been through fairly detailed QA now we can probably assume that the figures shown are enough now